### ISSUE TITLE
The range and rgbValue properties of Colour RGB resource should be arrays

### ISSUE THREAD

**Issue by qiuzhong** [siovene/smarthome-demo#18]
According to the [OCF specification](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/doc/oic.r.colour.RGB.json#L8), the range and rgbValue properties of Colour RGB resource should be arrays rather than strings.
The [SmartHome-Demo rgb_led.js](https://github.com/01org/SmartHome-Demo/blob/master/ocf-servers/js-servers/rgb_led.js) has complied it since [this commit](https://github.com/01org/SmartHome-Demo/commit/2c4c397aadd5b2162a30f5d71a1dce7cc96c7574). However, in the smart-home-control , the rgbValue is still treated as a string. See [controller.js](https://github.com/siovene/smart-home-control/blob/master/app/scripts/controllers.js#L94)
```
        $scope.sensor.data.properties.rgbValue = [
            $scope.color.r,
            $scope.color.g,
            $scope.color.b].join(',');
        OCFService.updateSensor($scope.sensor.data);
```
In that way, the rgbValue is a string. The adb log also shows the rgbValue is not an array.
```
01-21 03:17:04.545  5069  5101 D CordovaPluginOCF: Updating resource: {"id":{"deviceId":"coap:\/\/192.168.1.56:45153","resourcePath":"\/a\/rgbled"},"resourceTypes":["oic.r.colour.rgb"],"interfaces":["oic.if.baseline"],"mediaTypes":[],"properties":{"range":"[I@3bec5b1","rgbValue":"95,104,255","id":"rgbled"},"observable":true}
```

**Comment by siovene** [comment]
Thanks, it looks like the spec changed and I need to update the code to reflect that.

**Comment by siovene** [comment]
Fixed in v0.8.4.
